msg/vehicle_attitude_setpoint.msg

```diff
@@ -12,7 +12,11 @@ float32 yaw_sp_move_rate	# rad/s (commanded by user)
 float32[4] q_d			# Desired quaternion for quaternion control
 bool q_d_valid			# Set to true if quaternion vector is valid
 
-float32 thrust			# Thrust in Newton the power system should generate
+# For clarification: For multicopters thrust_x and thrust_y are usually 0 and thrust_z is the negative throttle demand.
+# For fixed wings thrust_x is the throttle demand and thrust_y, thrust_z will usually be zero.
+float32 thrust_x					# Throttle command in body x direction [-1,1]
```

Comment:
I strongly suggest that we use arrays to store vecotrs inside uORB messages like here:
https://github.com/PX4/Firmware/blob/master/msg/vehicle_local_position_setpoint.msg#L20

Reply:
@MaEtUgR Done!

Reply:
Perfect thanks!